stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -1257,47 +1255,6 @@ fileprivate func setAttrShapeList(
   }
 }
 
-@usableFromInline
-class _ThreadLocalState {
```

Comment:
`_ExecutionContext.global` is not thread-local. How do you plan to guarantee thread-safety if this no longer uses a thread-local state?

Reply:
Good to know that we need this to be thread-safe. Will fix.